docs/docs/guides/integrate/login-ui/typescript-repo.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Typescript Repository
+sidebar_label: Typescript Repository
+---
+
+To replace our current golang built login UI and showcase the use of our new resource and OIDC APIs, we've created the [Typescript Repository](https://github.com/zitadel/typescript).
+The Typescript Repository contains all TypeScript and JavaScript packages and applications you need to create your own ZITADEL Login UI.
+The repo implements the [OIDC standard](./oidc-standard)
+
+## Included Apps And Packages
+
+- `login`: The login UI used by ZITADEL Cloud, powered by Next.js
+- `@zitadel/server`: core components for establishing node client connection, grpc stub
+- `@zitadel/client`: core components for establishing web client connection, grpc stub
+- `@zitadel/react`: shared React utilities and components built with tailwindcss
+- `@zitadel/next`: shared Next.js utilities
+- `@zitadel/tsconfig`: shared `tsconfig.json`s used throughout the monorepo
+- `eslint-config-zitadel`: ESLint preset
+
+## Implemented features of OIDC Standard
+
```

Comment:
Can we add somehow that Login with username and password is already possible?

Reply:
I've added them under a new checkbox